src/config.ts

```diff
@@ -38,7 +38,7 @@ const getTargetClusterFromEnv = (): SolanaCluster => {
 		return targetCluster as SolanaCluster;
 	}
 
-	return SolanaCluster.LOCALNET;
+	return SolanaCluster.DEVNET;
```

Comment:
What's the reasoning behind falling back to DEVNET instead of LOCALNET?

IMO a fallback should be the safest option possible.